docs/content/preview/faq/operations-faq.md

```diff
@@ -105,35 +108,93 @@ You should see output like the following:
 (1 row)
 ```
 
 This confirms that the password for `John` will expire in approximately 4 hours.
+
+## Upgrade
+
+### What exactly does the upgrade with rollback feature do?
+
+The feature upgrades the binaries and enables new features that do not change the data format on disk.
+
+{{<lead link="../../yugabyte-platform/manage-deployments/upgrade-software-install/">}}
+For detailed information, see [Upgrade YugabyteDB](../../manage/upgrade-deployment/).
+{{</lead>}}
+
+### How long can a universe run after the upgrade but before the finalize step?
+
+A universe can run as long as needed while the application is being validated. However, it is recommended to finalize within 3 days. Operations like flag changes are disabled during the monitoring phase.
+
+### Are new YugabyteDB features disabled until the upgrade is finalized?
+
+New features that change the data format on disk are disabled. Because these data-format-changing features are new, they generally aren't being used in production environments yet, so their temporary disablement has no impact on current operations. 90% of code changes are validated from the new binary itself. The aim of rollback capabilities is to catch regressions to existing features or query plans and handle them quickly (by reverting to the previous binary without any data corruption).
+
+### Is it possible to run DDL operations during the upgrade (before finalize)?
+
+DDL operations are allowed for v2.20 to v2024.2 upgrades. DDL operations are only _blocked_ when the PostgreSQL version is upgraded, such as from v2024.2 (PostgreSQL 11) to v2025.1 (PostgreSQL 15).
+
+### Assuming rollback is not possible post-finalize, what happens if issues are observed in the application after finalization?
+
+YugabyteDB performs extensive testing to ensure there are no issues. For customers that are extremely risk averse, the recommendation is to upgrade and finalize a development environment and DR replicas (if any) first. After the data format on disk has changed, you cannot go back to old binaries—this is a technical limitation of any software that stores data to disk. The only way to rollback after finalize is to restore from a backup that was taken before the finalization (with loss of data).
+
+### How does the rollback feature interact with bidirectional xCluster setups?
+
+xCluster can only replicate from an old version to the new version. The target universe should be finalized before the source universe. If the source is finalized before the target, then xCluster automatically pauses itself (this only happens in certain versions that have an external data format change, such as v2024.2). For bidirectional setups, if writes are only happening on one side, then it's acceptable to upgrade and finalize the other side first and then upgrade the writing side. If both sides are taking writes, then both should be finalized at the same time; otherwise, replication in the new-to-old direction will not happen.
```

Comment:
```suggestion
xCluster can only replicate from an old version to the new version. You should finalize the target universe before the source universe. If the source is finalized before the target, then xCluster automatically pauses itself (this only happens in certain versions that have an external data format change, such as v2024.2).

For bidirectional setups, if writes are only happening on one side, then you can upgrade and finalize the other side first and then upgrade the writing side. If both sides are taking writes, then both should be finalized at the same time; otherwise, replication in the new-to-old direction will not happen.
```